fontforgeexe/bitmapview.c

```diff
@@ -734,24 +737,24 @@ static void BVExpose(BitmapView *bv, GWindow pixmap, GEvent *event ) {
     if ( bv->showgrid ) {
 	if ( bv->scale>2 ) {
 	    for ( i=bv->xoff+bv->scale; i<bv->width; i += bv->scale )
-		GDrawDrawLine(pixmap,i,0, i,bv->height,0xa0a0a0);
+		GDrawDrawLine(pixmap,i,0, i,bv->height,grid_color);
 	    for ( i=bv->xoff-bv->scale; i>0; i -= bv->scale )
-		GDrawDrawLine(pixmap,i,0, i,bv->height,0xa0a0a0);
+		GDrawDrawLine(pixmap,i,0, i,bv->height,grid_color);
 	    for ( i=-bv->yoff+bv->height-bv->scale; i>0; i -= bv->scale )
-		GDrawDrawLine(pixmap,0,i,bv->width,i,0xa0a0a0);
+		GDrawDrawLine(pixmap,0,i,bv->width,i,grid_color);
 	    for ( i=-bv->yoff+bv->height+bv->scale; i<bv->height; i += bv->scale )
-		GDrawDrawLine(pixmap,0,i,bv->width,i,0xa0a0a0);
+		GDrawDrawLine(pixmap,0,i,bv->width,i,grid_color);
 	}
-	GDrawDrawLine(pixmap,0,-bv->yoff+bv->height-0*bv->scale,bv->width,-bv->yoff+bv->height-0*bv->scale,0x404040);
+	GDrawDrawLine(pixmap,0,-bv->yoff+bv->height-0*bv->scale,bv->width,-bv->yoff+bv->height-0*bv->scale,guide_color);
 	GDrawDrawLine(pixmap,0,-bv->yoff+bv->height-bv->bdf->ascent*bv->scale,
-		bv->width,-bv->yoff+bv->height-bv->bdf->ascent*bv->scale,0x404040);
+		bv->width,-bv->yoff+bv->height-bv->bdf->ascent*bv->scale,guide_color);
 	GDrawDrawLine(pixmap,0,-bv->yoff+bv->height+bv->bdf->descent*bv->scale,
-		bv->width,-bv->yoff+bv->height+bv->bdf->descent*bv->scale,0x404040);
-	GDrawDrawLine(pixmap,bv->xoff+0*bv->scale,0, bv->xoff+0*bv->scale,bv->height,0x404040);
-	GDrawDrawLine(pixmap,bv->xoff+bv->bc->width*bv->scale,0, bv->xoff+bv->bc->width*bv->scale,bv->height,widthcol);
+		bv->width,-bv->yoff+bv->height+bv->bdf->descent*bv->scale,guide_color);
+	GDrawDrawLine(pixmap,bv->xoff+0*bv->scale,0, bv->xoff+0*bv->scale,bv->height,guide_color);
+	GDrawDrawLine(pixmap,bv->xoff+bv->bc->width*bv->scale,0, bv->xoff+bv->bc->width*bv->scale,bv->height,guide_color);
```

Comment:
These last two presumably defaulted to 0x000000 (from the definition in `charview.c` and could be changed separately from the other `guide_color` values. Is that likely to matter?

Reply:
Good point. Here's what the defaults look like prior to this change:
<details>
<summary>Screenshot</summary>

![bitmap_default](https://user-images.githubusercontent.com/1683303/66441234-ec31d100-ea03-11e9-95c5-91025512e215.png)

</details>

There's enough contrast against the background, I never even noticed the advance width was a different color from the other guide lines. So I don't think making them the same is a noticeable change.

Personally I don't need to see the advance as a different color. But in case someone does care, I can keep that as a separate option.